### PR TITLE
WordPress pt. 2:  Restore and update VIP Skeleton

### PR DESCRIPTION
**Pull request by montchr**

### CODE REVIEW

File: apps/nextjs/graphql/generated/index.tsx [hunk 1]
```diff
@@ -4335,6 +4359,35 @@ export type PostTypeLabelDetails = {
...
+  opengraphTitle?: Maybe<Scalars['String']>
+  opengraphType?: Maybe<Scalars['String']>
+  opengraphUrl?: Maybe<Scalars['String']>
+  readingTime?: Maybe<Scalars['Float']>
+  schema?: Maybe<SeoPostTypeSchema>
+  title?: Maybe<Scalars['String']>
+  twitterDescription?: Maybe<Scalars['String']>
+  twitterImage?: Maybe<MediaItem>
+  twitterTitle?: Maybe<Scalars['String']>
+}
```
Comment (montchr) [inline apps/nextjs/graphql/generated/index.tsx:4389]: This is magic!

File: apps/wordpress/composer.json [hunk 2]
```diff
@@ -48,11 +47,13 @@
...
+    "wpackagist-plugin/add-wpgraphql-seo": "^4.0",
+    "wpackagist-plugin/advanced-custom-fields": "^6.0",
     "wpackagist-plugin/query-monitor": "^3.11",
-    "wpackagist-plugin/debug-bar": "^1.1",
-    "wpackagist-plugin/wp-graphql": "^1.13",
-    "wpackagist-plugin/faustwp": "^0.8.1",
+    "wpackagist-plugin/wordpress-seo": "^19.0",
+    "wp-graphql/wp-graphql-acf": "^0.3",
+    "automattic/vip-bundle-decoupled": "dev-trunk",
+    "wpackagist-theme/twentytwentythree": "^1.0",
```
Comment (montchr) [inline apps/wordpress/composer.json:56]: TODO: Remove this and activate the internal theme instead.

File: apps/wordpress/config/application.php [hunk 3]
```diff
@@ -59,12 +59,30 @@
...
+// Base URL for the WordPress server.
+//
+// In a non-headless configuration, `WP_CONTENT_URL` would be defined according
+// to the value of `WP_HOME` (URL corresponding to the WordPress web server
+// root). In our headless configuration, however, we point `WP_HOME` to the
+// Next.js frontend, so its value no longer corresponds to the WordPress server.
+// While `WP_SITEURL` points to a location on the server, in our configuration
+// WordPress lives in a subdirectory `/wp`. To keep things simpler, we use an
+// additional environment variable to provide the base for `WP_SITEURL` and
+// `WP_CONTENT_URL`.
```
Comment (montchr) [inline apps/wordpress/config/application.php:71]: TODO: Explain this goofiness in the readme.

File: apps/wordpress/config/application.php [hunk 4]
```diff
@@ -98,6 +116,9 @@
 Config::define('LOGGED_IN_SALT', env('LOGGED_IN_SALT'));
 Config::define('NONCE_SALT', env('NONCE_SALT'));
 
+// Token for post previews in the headless client.
+Config::define('WP_CLIENT_PREVIEW_TOKEN', env('WP_CLIENT_PREVIEW_TOKEN'));
```
Comment (montchr) [inline apps/wordpress/config/application.php:120]: TODO: Remove? I don't think we need this anymore.

File: apps/wordpress/web/.htaccess [hunk 5]
```diff
@@ -1,7 +1,3 @@
-# SPDX-FileCopyrightText: 2022-2023 Temple University
-#
-# SPDX-License-Identifier: CC0-1.0
-
 # Allow rewrites for GraphQL endpoint
 
 <IfModule mod_rewrite.c>
```
Comment (montchr) [inline apps/wordpress/web/.htaccess:3]: TODO: Explain why this `.htaccess` file is actually essential. I think…?

File: apps/wordpress/web/app/themes/logan-center/style.css [hunk 6]
```diff
@@ -0,0 +1,11 @@
+/*
+Theme Name: Logan Center for Urban Investigative Reporting
+Theme URI: https://github.com/gregrickaby/nextjs-wordpress-theme
+Description: A theme to help turn WordPress into a headless CMS.
+Author: Greg Rickaby <greg@gregrickaby.com>
+Author URI: https://gregrickaby.com/
```
Comment (montchr) [inline apps/wordpress/web/app/themes/logan-center/style.css:6]: FIXME: Update author (license/credits are handled with the REUSE tooling)

File: apps/wordpress/web/app/themes/logan-center/theme.json [hunk 7]
```diff
@@ -0,0 +1,185 @@
...
+          "fontWeight": "700",
+          "lineHeight": "var(--wp--custom--typography--line-height--tiny)",
+          "fontSize": "var(--wp--custom--typography--font-size--extra-large)"
+        }
+      }
+    }
+  },
+  "customTemplates": [],
+  "templateParts": []
+}
```
Comment (montchr) [inline apps/wordpress/web/app/themes/logan-center/theme.json:185]: TODO: Sync with frontend actualities.

File: apps/wordpress/config/application.php (hunk 3 above)
Comment (montchr) [reply apps/wordpress/config/application.php:71]: Comment is enough -- it's not relevant outside of here.

File: apps/wordpress/web/app/themes/logan-center/theme.json (hunk 7 above)
Comment (montchr) [reply apps/wordpress/web/app/themes/logan-center/theme.json:185]: Later (nice to have)

**Comment by montchr** [comment]
Fixes for the comments forthcoming in #133

**Comment by astratagem** [comment]
Superseded by #140

**Pull request closed without merging**